packages/@o3r/eslint-config-otter/schematics/ng-add/linter/templates/tsconfig.eslint.json

```diff
@@ -1,9 +1,9 @@
 {
-  "extends": "./tsconfig.app",
+  "extends": "./tsconfig.json",
   "include": [
     ".eslintrc.js",
     "jest.config.js",
-    "src/**/*.ts",
-    "e2e-playwright/**/*.ts"
+    "**/src/**/*.ts",
+    "**/e2e-playwright/**/*.ts"
```

Comment:
Not sure it is required as the purpose of this file is to add file not included in build process